layouts/base.hbs

```diff
@@ -12,9 +12,8 @@
     <div class="container content">
       {{> body }}
     </div>
-    <div class="container-fluid footer">
```

Comment:
The Home Page didn't have a footer at all but just a list of socials, I'm not saying is not ok to add the footer to the homepage but it is a different design ;).